examples/models/deepseek-r1-distill-llama-8B/README.md

```diff
@@ -0,0 +1,72 @@
+# Summary
+This example demonstrates how to run [Deepseek R1 Distill Llama 8B](https://huggingface.co/deepseek-ai/DeepSeek-R1-Distill-Llama-8B) 3.8B model via ExecuTorch.
```

Comment:
Can you add that the distilled model architecture is exactly the same as Llama, so majority of the instructions here https://github.com/pytorch/executorch/blob/main/examples/models/llama/README.md is applicable.